requirements.txt

```diff
@@ -10,28 +10,28 @@ idna==2.8
 Inflector==2.0.12
 mock==3.0.5
 navmazing==1.1.6
-paramiko==2.5.0
-productmd==1.26
-pytest==5.4.3
-pytest-ibutsu==1.14.1
-pytest-services==1.3.1
-pytest-mock==1.10.4
+paramiko==2.7.2
+pexpect==4.8.0
+productmd==1.29
+pyotp==2.4.1
+pytest==6.1.2
```

Comment:
Why only `6.1.2` ?  In master we have 6.2.2 ... anything preventing to be on par with pytest in master ?

Reply:
No, just cherry-picked the commit that got us to this point. I'll bump this to 6.2.